src/bp/index.ts

```diff
@@ -76,6 +77,8 @@ try {
   require('dotenv').config({ path: path.resolve(process.PROJECT_LOCATION, '.env') })
   process.core_env = process.env as BotpressEnvironmentVariables
 
+  process.env.BP_REDIS_CHANNEL_PREFIX = process.env.BP_REDIS_CHANNEL_PREFIX || 'production'
```

Comment:
Environment variables are considered "read-only" and we shouldn't change them at runtime. We added `process.core_env` specially for that purpose. If you need to process an environment variable and change it afterwards, please add the "parsed" variable on the process directly.

A good example of that is `process.env.CLUSTER_ENABLED`. We parse that value using `yn` to know if it's true or false. Then we assign that value to `process.CLUSTER_ENABLED`. This way, we can always get the provided environment variable, and we have the processed value.